_docs/getting-started/design.md

```diff
@@ -11,7 +11,7 @@ last-modified: 2017-10-24
 
 
 
-<section>
+<section class="static-section">
```

Comment:
@araiford, is yours building ok without `markdown="1"` in here? Haven't tried building this yet, but I had to add that to get my stuff to render right on the other PR.

Reply:
@nimbinatus -- nope.  I had to add markdown="1".  If I missed it on a couple, I'll add it.